sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/datasources/persons/PersonsDbConnector.groovy

```diff
@@ -0,0 +1,27 @@
+package life.qbic.portal.sampletracking.datasources.persons
+
+import life.qbic.business.project.subscribe.PersonDataSource
+
+/**
+ * <b><short description></b>
+ *
+ * <p><detailed description></p>
+ *
+ * @since <version tag>
+ */
+class PersonsDbConnector implements PersonDataSource{
+    /**
+     * Returns the email that is found in the system for a user identified with the auth id
+     * Returns an empty string if nothing could be found.
```

Comment:
Do we need this interface and the subscriber table with its own name and e-mail address? Shouldn't all of the subscribers be in Liferay anyway? The way the db is structured we can only search for first and last name now, which could lead to problems.

Reply:
The db has an extra field for exactly this authentication provider.